Src/Newtonsoft.Json/JsonTextReader.cs

```diff
@@ -445,7 +445,7 @@ public override string ReadAsString()
         /// <summary>
         /// Reads the next JSON token from the stream as a <see cref="Byte"/>[].
         /// </summary>
-        /// <returns>A <see cref="Byte"/>[] or a null reference if the next JSON token is null. This method will return <c>null</c> at the end of an array.</returns>
+        /// <returns>A <see cref="Byte"/>[] or <c>null</c> if the next JSON token is null. This method will return <c>null</c> at the end of an array.</returns>
```

Comment:
Looks good to me. I'm happy sticking with `<c>null</c>` vs "JSON token is null" to differentiate the two concepts.